### DISCUSSION TITLE
Editing Playlists

### DISCUSSION THREAD

**Discussion by tanderson1992** [music-assistant/support#3291] [Q&A] [answered]
I'm new to Music Assistant so please pardon my ignorance. There are several mentions of easily editing playlists within MA, but I don't see any way to edit a playlist. Items can be added or removed, but I can't change the order other than sorting a different way. Is it possible to move songs around in a playlist? For example, my wife likes to add new songs to the top of the playlist and for a few days play that list unshuffled to be sure the new songs are heard. As time goes by she's happy to hear songs randomly, but sometimes she wants the songs in the order she wants them.

**Comment by OzGav** [comment, marked as answer]
I am not sure where you have seen "editing" of playlists mentioned. "Creating" yes. There is an item on the backlog to allow reordering of playlists. There will be quite some time before that is implemented though, unless more devs step up.

**Reply by TicoM1** [reply]
This one? 
https://github.com/orgs/music-assistant/projects/2/views/1?filterQuery=Playlist&pane=issue&itemId=62585784
Would at least take a look. Is there any existing discussion about implementation details? Can you point me in a direction on where to start having a look at the code? Completely new to the project...

**Reply by OzGav** [reply]
Yes that is it. You should come to Discord and ask in the devs-support chat

**Comment by tanderson1992** [comment]
I saw this in a 2022 post, "The easiest way is to just use MA to edit your playlists," but I see that wasn't from a dev. Thanks for the quick reply. I'll keep ordering them in the m3u. I really like MA so far and expect to love it once I figure out what the heck I'm doing!

**Reply by OzGav** [reply]
2022 was version 1 of MA so longer valid.
